containers/Connector/Connector.tsx

```diff
@@ -50,7 +50,7 @@ const useConnector = () => {
 			const useOvm = getIsOVM(networkId);
 
 			synthetix.setContractSettings({
-				networkId: useOvm ? 1 : networkId,
```

Comment:
Maybe a util function for networkID because we reuse this a few times

Reply:
Say more, the edit here actual removes that conditional :)

Reply:
You're right. I'm not sure why I was looking at removed lines. Everything looks good!